lms/djangoapps/verify_student/services.py

```diff
@@ -13,10 +13,40 @@
 from student.models import User, CourseEnrollment
 from lms.djangoapps.verify_student.models import VerificationCheckpoint, VerificationStatus, SkippedReverification
 
+from .models import SoftwareSecurePhotoVerification
 
 log = logging.getLogger(__name__)
 
 
+class VerificationService(object):
+    """
+    Learner verification XBlock service
+    """
+
+    def get_status(self, user_id):
+        """
+        Returns the user's current photo verification status.
+
+        Args:
+            user_id: the user's id
+
+        Returns: one of the following strings
+            'none' - no such verification exists
+            'expired' - verification has expired
+            'approved' - verification has been approved
+            'pending' - verification process is still ongoing
+            'must_reverify' - verification has been denied and user must resubmit photos
+        """
+        user = User.objects.get(id=user_id)
+        return SoftwareSecurePhotoVerification.user_status(user)
```

Comment:
I know I complained about this before, but I checked, and we don't seem to have a better way of getting this information, alas.